composites/OnboardingWizard/onboarding-wizard.scss

```diff
@@ -44,6 +44,23 @@ body {
   box-sizing: border-box;
   width: 100%;
 
+  &--header {
+    text-align: center;
+
+    h1 {
+      color: $color_pink_dark;
+      margin: 0 0 -16px;
+      font-size: 1.2em;
+      letter-spacing: 0.03em;
```

Comment:
computes to 0.576px but I guess we can live with this?
Worth noting the actual rendering of this font in WordPress will be slightly different because here in the test environment the font is Arial, while WordPress uses the system fonts stack:
`-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif`